### PR TITLE
Back 1219 poll dto caching patch2.8.1 h3

### PR DESCRIPTION
**Pull request by RomanWlm**

### CODE REVIEW

File: src/it/scala/co/ledger/wallet/daemon/api/AccountsApiTest.scala [hunk 1]
```diff
@@ -181,7 +181,7 @@ class AccountsApiTest extends APIFeatureTest {
     assert(walPoolView.findValue("wallet_count").asInt() == 3)
     info(s"Pool is = $pool and $walPoolView")
     deletePool(poolName)
-    getPool(poolName, Status.NotFound)
+    getPool(poolName, Status.BadRequest)
```
Comment (mbasire-ledger) [inline src/it/scala/co/ledger/wallet/daemon/api/AccountsApiTest.scala:184]: Do the endpoint change ?

File: tooling/database/postgres/dbs.sh [hunk 2]
```diff
@@ -22,7 +22,7 @@ $command -h $host -p $port wallet_daemon -w -U $user
 echo "     |--> wallet_daemon Done"
 
 # Databases used in IT tests
-tests=(transactionsExcludeUTXOTest transactionsPartialTest currency_pool test_pool test_pool1 op_pool op_pool_mal info_pool exist_pool fresh_addresses_pool list_pool account_pool delete_pool delete_account balance_pool pool_1 pool_2 pool_3 random_pool multi_pool_mal transactionsCreation4Test ledger anotha_pool same_pool this_pool my_pool your_pool wallet_pool duplicate_pool empty_pool multi_pool random POOL_NAME myPool test_wallet getUtxo_pool )
+tests=(transactionsExcludeUTXOTest transactionsPartialTest currency_pool test_pool test_pool1 op_pool op_pool_mal info_pool exist_pool fresh_addresses_pool list_pool account_pool delete_pool delete_account balance_pool pool_1 pool_2 pool_3 random_pool multi_pool_mal transactionsCreation4Test ledger anotha_pool same_pool this_pool my_pool your_pool wallet_pool duplicate_pool empty_pool multi_pool random POOL_NAME myPool test_wallet getUtxo_pool pool_name )
```
Comment (mbasire-ledger) [inline tooling/database/postgres/dbs.sh:25]: you added a pool "pool_name" and there is still "POOL_NAME" but in some test cases, _POOL_NAME_ became _pool_name_, is that a mistake ?

**Review by mbasire-ledger — APPROVED** [review]

File: src/it/scala/co/ledger/wallet/daemon/api/AccountsApiTest.scala (hunk 1 above)
Comment (RomanWlm) [reply src/it/scala/co/ledger/wallet/daemon/api/AccountsApiTest.scala:184]: Yes, we return bad request now when we hit an non existing endoint

File: tooling/database/postgres/dbs.sh (hunk 2 above)
Comment (RomanWlm) [reply tooling/database/postgres/dbs.sh:25]: Something to remove, actually the endpoint that delete pool is loosing future that asking for pool deletion. So we return before the pool is effectively deleted. Indeed POOL_NAME and pool_name is same for PG. (was an deseperate attemps after having found "database wdtest_pool_name does not exists" I remove POOL_NAME.

**Pull request merged**